framework/src/modules/governance/stores/proposals.ts

```diff
@@ -25,4 +32,101 @@ export interface ProposalsStoreData {
 	description: ProposalDescription;
```

Comment:
```suggestion
	description: ProposalDescription;
	data: Buffer;
	status: number;
```

Reply:
Good catch. `status` property should actually be of `ProposalStatus` type.